File: IT.Web/src/app/models/transaction-grid.model.ts
export interface TransactionGridModel {
  id: number;
  transactionDate: Date;
  amount: number;
  interest150: number;
  interest125: number;
  interest100: number;
}

export interface TransactionGridUiModel {
  id: number;
  transactionDate: string | null;
  amount: string | null;
  interest150: string | null;
  interest125: string | null;
  interest100: string | null;
}

File: IT.Api/DataTransferModels/TransactionGridModel.cs
using System;

namespace IT.Api.DataTransferModels;

public class TransactionGridModel
{
    public int Id { get; set; }
    public DateTime TransactionDate { get; set; }
    public int Amount { get; set; }
    public int Interest150 { get; set; }
    public int Interest125 { get; set; }
    public int Interest100 { get; set; }
}
